scalardb_fdw/expected/scalardb_fdw.pg15.out

```diff
@@ -895,3 +895,39 @@ explain verbose select * from postgresns_test where p_pk = 1 order by p_ck1 ASC,
          ScalarDB Scan Attribute: ("p_pk" "p_ck1" "p_ck2" "p_boolean_col" "p_int_col" "p_bigint_col" "p_float_col" "p_double_col" "p_text_col" "p_blob_col")
 (10 rows)
 
+-- Columns that are required to evaluate WHERE clause locally at PostgreSQL side must be returned from the remote server
+-- - ForeignScan on cassandrans_test must return c_boolean_col
+explain verbose select p_pk from postgresns_test inner join cassandrans_test on p_pk = c_pk where c_boolean_col;
+                                        QUERY PLAN                                         
+-------------------------------------------------------------------------------------------
+ Merge Join  (cost=303.89..632.27 rows=21404 width=4)
```

Comment:
Hmm, right. It might change between the versions. One possible naive approach is to check the Postgres version when the tests start and abort the tests if the version differs from what we expect. 

Also, it would be helpful to fix the version of Postgres to use for the testing using a tool like [mise](https://mise.jdx.dev/). (I use it for the local testing, but don't include the `.mise.toml` in the repository so far.)

But I haven't encountered such a situation so far, so let's leave it as it is for now. Once we see the test failing due to the difference in the cost, we will consider how to solve it then.